nautobot/extras/models/__init__.py

```diff
@@ -1,4 +1,11 @@
-from .change_logging import ChangeLoggedModel, ObjectChange
+from .change_logging import ChangeLoggedModel, ObjectChange  # isort:skip
+from .approvals import (  # isort: skip
+    ApprovalWorkflow,
+    ApprovalWorkflowInstance,
+    ApprovalWorkflowStage,
+    ApprovalWorkflowStageInstance,
+    ApprovalWorkflowStageInstanceResponse,
+)
```

Comment:
We are going to have to import ApprovalWorkflow models later than ChangeLoggedModel because of a circular import issue.